cls/SourceControl/Git/Utils.cls

```diff
@@ -828,8 +854,28 @@ ClassMethod ImportItem(InternalName As %String, force As %Boolean = 0, verbose A
 ClassMethod ListItemsInFiles(ByRef itemList, ByRef err) As %Status
 {
     #define DoNotLoad 1
-    set res = $system.OBJ.ImportDir(..TempFolder(),"*.xml","-d",.err,1, .itemList, $$$DoNotLoad)
     
+    set mappingFileType = $order($$$SourceMapping(""))
```

Comment:
Another thought: we don't require explicit mappings if you're loading via the package manager. Might be worth considering the behavior of ListItemsInFiles in this case as well. (Though it's much trickier... Happy to just not worry about it for now.)